### DISCUSSION TITLE
Durable Functions - Forbidden error message

### DISCUSSION THREAD

**Discussion by isAarya** [Azure/azure-functions-durable-extension#2243] [Q&A]
Hi,

I am learning Azure Durable Functions.

But when I use the Azure Durable Orchestrator Function template as a start in Visual Studio 2022.

I receive the following error message.

The listener for function 'Function1' was unable to start.
The listener for function 'Function1' was unable to start. DurableTask.AzureStorage: Forbidden. Microsoft.WindowsAzure.Storage: Forbidden.

Not sure what setting I am missing which is causing this error message.

Has anyone seen a similar error.

FYI I am using latest Visual 2022 with Azurite running for local storage. I am also behind VPN

And the local.settings.json file is as follows,

"IsEncrypted": false,
"Values": {
"AzureWebJobsStorage": "UseDevelopmentStorage=true",
"FUNCTIONS_WORKER_RUNTIME": "dotnet",
"AzureWebJobsSecretStorageType": "files"
}

Thanking you in advance.

Kind regards

**Comment by davidmrdavid** [comment]
Hi @isAarya, thank you for reaching out.

Hmm, I don't know on top of mind what could cause this issue. 
A few questions / requests:
(1) You mentioned that you're behind a VPN. I don't think that should affect anything, but have you confirmed just as a sanity check that this fails even if you're not behind a VPN? There should be no reason for VPN usage in local development as far as the starter templates are concerned.
(2) If the issue continues, could you please attach a `.zip`'ed reproducer? That would help us speed up this investigation. Thanks!
